### PR TITLE
CI: try explicitly including msvc_runtime

### PR DESCRIPTION
**Pull request by tacaswell**
Might solve #6115

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ install:
   - cmd: copy "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\vcvars64.bat" "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\amd64\vcvarsamd64.bat"
 
   # same things as the requirements in ci/conda_recipe/meta.yaml
-  - cmd: conda create -y -q -n test-environment python=%PYTHON_VERSION% pip setuptools numpy python-dateutil freetype msinttypes tk pyparsing pytz tornado libpng zlib pyqt cycler nose mock
+  - cmd: conda create -y -q -n test-environment python=%PYTHON_VERSION% pip setuptools numpy python-dateutil freetype msinttypes tk pyparsing pytz tornado libpng zlib pyqt cycler nose mock msvc_runtime
```
Comment (JanSchulz) [inline appveyor.yml]: IMO this isn't necessary, the fix is the `libpng` (without `# [unix]`) line below?

Comment (JanSchulz) [inline appveyor.yml]: ok, strike that this seems to be fixing the 3.3 error, I looked only at the 27 one?

**Comment by JanSchulz** [comment]
IMO this should be merged... Maybe the conda ppl can shed some light why this is only now failing... OR if the msvc dependency it the right thing to do...

**Comment by jenshnielsen** [comment]
Ok lets merge this. If it can be done in a more clean way we can change it again

**Pull request merged**